shm/tests/test.cpp

```diff
@@ -0,0 +1,32 @@
+#include "alcohol.hpp"
+#include "gtest/gtest.h"
+
+class AlcoholTest : public ::testing::Test {
+protected:
+    AlcoholTest() : alco1_("Whisky", 5, 200, 48) {}
+    Alcohol alco1_;
+};
+
+TEST_F(AlcoholTest, IncOperatorShouldAddToAmount) {
+    auto amountValue = 5;
+    auto oldAmount = alco1_.getAmount();
+
+    alco1_ += amountValue;
+    ASSERT_EQ(alco1_.getAmount(), amountValue + oldAmount);
+}
+
+TEST_F(AlcoholTest, DecOperatorShouldSubstractFromAmount) {
+    auto amountValue = 4;
+    auto oldAmount = alco1_.getAmount();
+
+    alco1_ -= amountValue;
+    ASSERT_EQ(alco1_.getAmount(), oldAmount - amountValue);
+}
+
+TEST_F(AlcoholTest, DecOperatorShouldBeNowLowerThanZero) {
```

Comment:
I think there is a typo in a test case name :)
```suggestion
TEST_F(AlcoholTest, DecOperatorShouldNotBeLowerThanZero) {
```

Nice tests BTW, descriptive naming.